sw/device/silicon_creator/rom/e2e/BUILD

```
@@ -53,8 +55,8 @@ SLOTS = {
     "b": "0x80000",
 }
 
-opentitan_flash_binary(
-    name = "empty_test_slot_a",
+[opentitan_flash_binary(
+    name = "empty_test_slot_{}".format(slot),
```

Nit. Just a matter of style.
```suggestion
    name = f"empty_test_slot_{slot}",
```

---

unfortunately bazel does not support f-strings.